cmd/podman/shell_auto_completion.go

```diff
@@ -0,0 +1,68 @@
+package main
+
+import (
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+const (
+	bashAuthoCompletionDescription = `To load completion run
```

Comment:
I need to correct the typo here `Autho` instead of `auto` - I am playing too much with authorised transaction ;)

Still WIP, don't have a lot of spare time but should be able to finish this off by the end of the week.